README.md

```diff
@@ -63,12 +63,12 @@ The orignal v1.0 dev roadmap [is here ](https://github.com/orgs/openebs/projects
 
 > | Name | Version |
 > | :--- | :--- |
-> | K8S | 1.20+ |
+> | K8S | 1.23+ |
 > | Distro | Alpine, Arch, CentOS, Debian, Fedora, NixOS, SUSE, RHEL, Ubuntu |
```

Comment:
Not part of this change but have we tested all these Distros ?

Reply:
I don't know about the older versions, but the latest few versions haven't been tested.